wallet/utxos_test.go

```diff
@@ -129,6 +130,84 @@ func TestFetchOutpointInfo(t *testing.T) {
 	require.Equal(t, int64(0-testBlockHeight), confirmations)
 }
 
+// TestFetchOutpointInfoErr checks when the wallet cannot find an output, a
+// proper error is returned.
+func TestFetchOutpointInfoErr(t *testing.T) {
+	t.Parallel()
+
+	w, cleanup := testWallet(t)
+	defer cleanup()
+
+	// Create an address we can use to send some coins to.
+	addr, err := w.CurrentAddress(0, waddrmgr.KeyScopeBIP0084)
+	require.NoError(t, err)
+	p2shAddr, err := txscript.PayToAddrScript(addr)
+	require.NoError(t, err)
+
+	// Create a tx that has two outputs - output1 belongs to the wallet,
+	// output2 is external.
+	output1 := wire.NewTxOut(100000, p2shAddr)
```

Comment:
Hmm why do they they pay to the same addr thought therefore we created addr above ?

Reply:
doesn't matter but still better to have a proper logic in.

Reply:
they are dummy values - techinically they should be the same so we know the result doesn't depend on the pkscript, but again I don't think it matters.